pkg/git/GoGitSDKManager.go

```diff
@@ -2,22 +2,25 @@ package git
 
 import (
 	"fmt"
+	"go.uber.org/zap"
 	"gopkg.in/src-d/go-git.v4"
 	"gopkg.in/src-d/go-git.v4/config"
 	"gopkg.in/src-d/go-git.v4/plumbing"
 	"os"
 )
 
-type GoGitSDKManager interface {
-	GitManager
-}
+//	type GoGitSDKManager interface {
+//		GitManager
+//	}
 type GoGitSDKManagerImpl struct {
-	*GitManagerBaseImpl
+	GitManagerBase
```

Comment:
interface here but impl in cliManager. why?

Reply:
can we make interfaces in both places?

Reply:
Used interface in cliManager as well